variables.tf

```diff
@@ -817,3 +817,17 @@ EOF
     error_message = "Invalid value for var.pipeline_location. Supported values: URL path. Example: https://github.com/dfds/terraform-aws-rds/actions/workflows/qa.yml"
   }
 }
+
+variable "automation_initiator_location" {
+  description = <<EOF
+    Specify the URL to the repo of automation script.
+    Valid Values: URL to repo. Example: `"https://github.com/dfds/terraform-aws-rds"`
+    Notes: This set the dfds.automation.initiator.location tag. See recommendations [here](https://wiki.dfds.cloud/en/playbooks/standards/tagging_policy).
+  EOF
+  type        = string
+  default     = null
+  validation {
+    condition     = var.automation_initiator_location == null || can(regex("(https:\\/\\/www\\.|http:\\/\\/www\\.|https:\\/\\/|http:\\/\\/)?[a-zA-Z0-9]{2,}(\\.[a-zA-Z0-9]{2,})(\\.[a-zA-Z0-9]{2,})?\\/[a-zA-Z0-9]{2,}", var.automation_initiator_location))
+    error_message = "Invalid value for var.automation_initiator_location. Supported values: URL path. Example: https://github.com/dfds/terraform-aws-rds"
```

Comment:
I think we need to extend the error message for ADO too since it won't work with `?` in the URL.

Reply:
hmm do have an example of such url that I can use to test?

Reply:
Example sent on Slack